droid-help/src/main/resources/Web pages/Command line control.html

```diff
@@ -230,6 +241,20 @@ <h3>
 <p>
     <i>Note:</i> if you specify "stdout" as the filename for the -o option, it will also write to the console.  But it's easier to simply not specify the -o option.
 </p>
+<p>
+    To output JSON results to a file rather than CSV, you can use the <b>--json</b> option:
```

Comment:
Should this be "To output results to a JSON rather than CSV..." or are you actually saying it produces JSON results that can be output to a CSV/JSON?

Reply:
No, you're right, this doesn't make sense. I think what I meant was
`To output results as JSON instead of CSV, you can use the <b>--json</b> option:`